### PR TITLE
Added confirmation dialog for delete action of datacenter objects and changed color of restore/rollback buttons

### LINKED ISSUE #135: Add confirmation dialog to all delete/destroy actions
Add confirmation dialog to following actions in GUI:

- Delete Compute Node (compute node -> edit modal)
- Delete DNS Domain
- Delete User Group
- Delete User
- Delete ISO Image
- Delete Image
- Delete Network
- Delete Datacenter

Change color of restore/rollback button, delete confirmation will be discussed in #203:

- Delete/Rollback snapshot
- Delete/Restore backup

### PR DESCRIPTION
**Pull request by dn0**
Closes #135

### CODE REVIEW

File: gui/templates/gui/vm/backup_restore_modal.html [hunk 1]
```diff
@@ -24,8 +24,8 @@
   <div class="modal-footer">
     <div class="inner-well">
       <a class="vm_modal_no button blue no-ajax" data-dismiss="modal">{% trans "Cancel" %}</a>
+      <a class="vm_modal_yes_force button gold no-ajax">{% trans "Restore backup" %}</a>
       <a class="vm_modal_yes button red no-ajax">{% trans "Delete backup" %}</a>
```
Comment (secult) [inline gui/templates/gui/vm/backup_restore_modal.html]: I would put confirm also here to be consistent across all delete buttons

File: gui/templates/gui/vm/snapshot.html [hunk 2]
```diff
@@ -93,8 +93,8 @@
       <span class="pull-left"><a href="#" id="image_snapshot_link" class="btn btn-small btn-inverse no-ajax obj_add" data-prefix="img">{% trans "Create Image" %}</a></span>
       {% endif %}
       <a class="vm_modal_no button blue no-ajax" data-dismiss="modal">{% trans "Cancel" %}</a>
+      <a class="vm_modal_yes_force button gold no-ajax">{% trans "Rollback snapshot" %}</a>
       <a class="vm_modal_yes button red no-ajax">{% trans "Delete snapshot" %}</a>
```
Comment (secult) [inline gui/templates/gui/vm/snapshot.html]: I would put confirm also here to be consistent across all delete buttons

File: gui/templates/gui/vm/backup_restore_modal.html (hunk 1 above)
Comment (dn0) [inline gui/templates/gui/vm/backup_restore_modal.html]: Let's move this discussion into #203

File: gui/templates/gui/vm/snapshot.html (hunk 2 above)
Comment (dn0) [inline gui/templates/gui/vm/snapshot.html]: Lets move this discussion into #203

**Pull request merged**